app/services/force/custom_api/rental_assistance_service.rb

```diff
@@ -7,6 +7,20 @@ class RentalAssistanceService < Force::Base
       def destroy(id)
         @client.delete("/services/apexrest/RentalAssistance/#{id}")
       end
+
+      def delete_by(record, id)
```

Comment:
instead of record can we call this param `field_name`? I was confused at what this method was doing until I saw the call

Reply:
done